src/index.js

```diff
@@ -1,11 +1,10 @@
 import './style.css';
```

Comment:
Great job @David-Kasilia 🤝

- [ ] Kindly import the images correctly into your project. The images are lacking in your design
![imagess](https://user-images.githubusercontent.com/63917725/183873938-d2f561d0-0c97-4150-95e4-9a814ff70ca9.PNG)

Reply:
@1GORDON Hello, I would like to request your help in implementing this because I have tried linking the images and the CSS file again and it still not loading the 9images as required.
Is this because of the file loader dependency for webpack?

Reply:
Hello @David-Kasilia, Kindly take a look at the [webpack getting started](https://webpack.js.org/guides/getting-started/) documentation. Go through the [asset management](https://webpack.js.org/guides/asset-management/) properly.

Reply:
@1GORDON Thanks alot for your help. I went through it again and used the appropriate method it is now functional.You may confirm that if you would like